.github/workflows/CI.yml

```diff
@@ -0,0 +1,45 @@
+name: CI
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+    tags: '*'
+jobs:
+  test:
+    name: Julia ${{ matrix.version }} - ${{ matrix.os }} - ${{ matrix.arch }} - ${{ github.event_name }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        version:
+          - '1.0'
+          - '1'
```

Comment:
```suggestion
          - '1'
          - nightly
```
?

Reply:
That would be fine. This package is unlikely to have nightly-specific failures on a frequent basis. I'd approve that change.

Reply:
(FYI I can't merge PRs here.)

Reply:
Nor can I give you that power, unfortunately. (I'm a collaborator but don't have full administrative privileges.)